libnavigation-core/src/test/java/com/mapbox/navigation/core/MapboxNavigationTest.kt

```diff
@@ -1195,11 +1195,31 @@ internal class MapboxNavigationTest : MapboxNavigationBaseTest() {
         assertFalse(mapboxNavigation.getRerouteController() === oldRerouteController)
     }
 
+    @Test
```

Comment:
NIT

Normally is good to structure the tests following the triple A 👀 https://speakerdeck.com/guardiola31337/elegant-unit-testing-mobilization-2016?slide=40 http://wiki.c2.com/?ArrangeActAssert

```kotlin
@Test
fun ignoreEnhancedLocationUpdates() {
    // Arrange
    
    // Act
    
    // Assert
}
```

Note the empty lines between Arrange / Act and Act / Assert.

It's cosmetics, although improves readability IMO.

The same thing applies to some other tests.